src/actors/redis-generator/redis-generator.py

```diff
@@ -0,0 +1,70 @@
+import sys
+import json
+import shlex
+
+from subprocess import Popen, PIPE
+
+
+def _execute(cmd):
```

Comment:
check_output + subprocess.CalledProcessError  can be a more sexy solution for this

Reply:
That's right, thank you!